src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -71,8 +62,9 @@ public TestCaseAttribute(params object[] arguments)
         /// <param name="arg"></param>
         public TestCaseAttribute(object arg)
         {
-            this.RunState = RunState.Runnable;			
-            this.Arguments = new object[] { arg };
+            RunState = RunState.Runnable;			
+            Arguments = new object[] { arg };
+            Properties = new PropertyBag();
```

Comment:
In the future, it might be nice to chain each of these constructors to a private default constructor that sets the RunState and creates the PropertyBag in one place rather than having to remember to repeat in every constructor.